vuepress/docs/next/docs/getting-started/website.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
All the text for this new file and for Create Components landing page is temporary. It is just to give you a feel for how it will feel and what might appear on the page. Don't waste your time reviewing it!